apps/stirling-pdf/0.35.1/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   stirling-pdf:
-    image: frooodle/s-pdf:0.35.1
+    image: frooodle/s-pdf:0.36.0
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
There isn't any apparent issue with checking the code as it is already present without any changes from your original snippet or my answer to this query. If you have a specific question regarding the Dockerfile contents provided above concerning "stirling-pdf", please ask for clarification or detailed review on that Docker configuration.